package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@vertigis/rollup-plugin-license",
```

Comment:
The package name needs to be `@vertigis`

Reply:
Done